addons/io_scene_gltf2/blender/exp/gltf2_blender_gather.py

```diff
@@ -18,6 +18,7 @@
 from io_scene_gltf2.blender.exp import gltf2_blender_gather_nodes
 from io_scene_gltf2.blender.exp import gltf2_blender_gather_animations
 from io_scene_gltf2.blender.exp.gltf2_blender_gather_cache import cached
+from io_scene_gltf2.blender.exp import gltf2_blender_generate_extras
 
```

Comment:
add this import for my suggestion below
```suggestion
from io_scene_gltf2.blender.exp import gltf2_blender_export_keys

```